app/models/spree/order_decorator.rb

```diff
@@ -45,8 +48,22 @@ def commit_avatax_invoice(doc_type)
             :discounted => true,
             :item_code => 'SHIPPING'
         )
+
         invoice_lines << invoice_line
 
+        charge_line = Avalara::Request::Line.new(
+            :line_no => (line_count + 1).to_s,
+            :destination_code => '1',
+            :origin_code => '1',
+            :qty => 1,
+            :amount => charge.to_s,
+            :tax_code => 'FR',
+            :discounted => true,
+            :item_code => 'CHARGE'
+        )
+
+        invoice_lines << charge_line if charge > 0
+
```

Comment:
Avalara calls take time and cost money. I don't know if _Avalara::Request::Line.new_  makes an API call or not.
Should we be calling this all the time or just if charge > 0?